tests/steps/application_v2_steps.py

```diff
@@ -404,21 +403,26 @@ def remember_app_id(context):
     context.app_ids.append(app_id)
 
 
+def wait_for_algod_transaction_processing_to_complete():
+    """
+    wait_wait_for_algod_transaction_processing_to_complete is a Dev mode helper method that's a rough analog to `context.app_acl.status_after_block(last_round + 2)`.
```

Comment:
I can change this after merge so we don't run the CI again:
```suggestion
    wait_for_algod_transaction_processing_to_complete is a Dev mode helper method that's a rough analog to `context.app_acl.status_after_block(last_round + 2)`.
```